### ISSUE TITLE
Zenmuse P1 stream not working 

### ISSUE THREAD

**Issue by lorenzave** [dji-sdk/Onboard-SDK-ROS#458]
Hi Support!

We are trying to integrate to the P1 to use it in a similar fashion as the H20T but we get no h264stream or frames.

## Setup
- M300 fw: 02.02.0301
- P1 fw: 02.02.03.02
- OSDK: 4.1.0
- OSDK-ROS: 4.1.0

## Description

Call service to enable main camera stream:
`rosservice call /dji_osdk_ros/setup_camera_h264 "request_view: 0 start: 1"`

and the vehicle node logs report the camera is not mounted.
Output:
```
[2273189.694]STATUS/1 @ getCameraList, L218: Request start pushing camera info ack = 0
[2273190.715]STATUS/1 @ getCameraList, L235: Request end pushing camera info ack = 0
[2273190.716]STATUS/1 @ getCameraList, L236: mounted	cam0:0 cam2:0 cam3:0
[2273190.716]STATUS/1 @ getCameraList, L237: type   	cam0:0 cam2:0 cam3:0
[2273190.716]ERRORLOG/1 @ startH264Stream, L459: camera[0] is not mounted
```

Similar trying to get decoded frames (this one reports an error in the service response):
```
rosservice call /dji_osdk_ros/setup_camera_stream "cameraType: 1 start: 1"
ERROR: service [/dji_osdk_ros/setup_camera_stream] responded with an error: b''
```
Vehicle node log output:
```
STATUS/1All components for decoding initialized ...
STATUS/1User callback thread created successfully!
STATUS/1****** Decoder Callback Thread Start ******
[2273322.736]STATUS/1 @ getCameraList, L218: Request start pushing camera info ack = 0
[2273323.755]STATUS/1 @ getCameraList, L235: Request end pushing camera info ack = 0
Decoder Callback Thread: Get image time out
[2273323.756]STATUS/1 @ getCameraList, L236: [2273323.756]STATUS/1 @ getCameraList, L237: [2273323.756]ERRORLOG/1 @ startH264Stream, L459: camera[0] is not mounted
```

So basically there is no way we can get images or an h264 stream right now with this setup. Is it because the current versions of OSDK don't support the P1?

I hope to hear from you soon. :+1:
